### DISCUSSION TITLE
EPERM not permitted to rename

### DISCUSSION THREAD

**Discussion by connermurphy** [parcel-bundler/parcel#5353] [General]
I'm using ParcelJS 2.0.0-nightly.447 and I keep getting the below error. I run **npm run **dev** and it's fine, but as soon as I make a change to my code and save the CMD Prompt throws this error.

**Error: EPERM: operation not permitted, rename 'F:WebsiteDevelopmentMarchetCodedistindex.html.191289732' -> 'F:WebsiteDevelopmentMarchetCodedistindex.html'**

**Comment by DeMoorJasper** [comment]
There's not much to go on with this... I assume it's a permission issue with your folder 🤷‍♂️

**Reply by mischnic** [reply]
Has been reported before: https://github.com/parcel-bundler/parcel/issues/4955

**Reply by connermurphy** [reply]
It's just decided to fix itself now... I'm genuinely stumped because I literally tried everything possible.

**Reply by connermurphy** [reply]
And now it's broke again. This is genuinely confusing.
